### ISSUE TITLE
The number of returned indeces of 'get_noise_indices' differes from num_label_errors

### ISSUE THREAD

**Issue by filippoBUO** [cleanlab/cleanlab#113]
Hi,

Even if I put frac_noise=1 in the function 'get_noise_indices' the length of the vector of noise_indices is different from the returned number from the function num_labels_errors. The num_labels_erros I found to be the sum of all the elements of the confident_joint matrix minus the sum of its diagonal elements as expected.

Could you explain me why there is this difference? 

Thank you!

**Comment by anishathalye** [comment]
Thank you for pointing out that the documentation doesn't explain this discrepancy. This was fixed in 8f9f3f53801c1100a25b458255e134877844e9d0; see the rendered docs here: https://docs.cleanlab.ai/master/cleanlab/count.html#cleanlab.count.num_label_issues.

> This method is more accurate than sum(find_label_issues()) because its computed using only the Trace(joint), ignoring all off-diagonals (used by find_label_issues and harder to estimate). Here we sum over only diagonal elements in the joint (which have more data are more constrained, and therefore easier to compute).
> 
> tl;dr - Use this method to get the most accurate estimate of number of label issues when you don’t need the indices of the label issues. This is the canonical way to find errors simply by combining a ranking/scoring function from rank.py with num_label_issues().
